icu4j/main/classes/core/src/com/ibm/icu/message2/Formatter.java

```diff
@@ -10,7 +10,7 @@
  * that can be used from {@link MessageFormatter}.
  *
  * @internal ICU 72 technology preview
- * @deprecated This API is for ICU internal use only.
+ * @deprecated This API is for technology preview only.
```

Comment:
Changed "This API is for ICU internal use only." to "This API is for technology preview only." for technology preview APIs.

Reply:
Thanks! And sorry that I missed those :-(
FYI  @mihnita